lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -28,6 +29,9 @@ const (
 var (
 	portLayerClient     *client.PortLayer
 	portLayerServerAddr string
+
+	// TODO(jzt): should this be defined in backends/image.go?
+	layerCache = NewLayerCache()
```

Comment:
I would really like to have some kind of a getter function to get this.  Since this file is separate but in the same package, addressing this via direct name might be confusing to some people because they don't know where this var is.  Same as we do with PortLayerClient.